### ISSUE TITLE
DataFrames.CategoricalValue not displaying nicely

### ISSUE THREAD

**Issue by gszep** [fonsp/Pluto.jl#807]
![Screenshot from 2020-12-25 11-19-12](https://user-images.githubusercontent.com/17977117/103132017-69963f80-46a3-11eb-89fa-939aaffff571.png)

**Comment by pankgeorg** [comment]
Hey! Happy holidays!
How would you like that to look? any suggestions?

**Comment by PaulToronto** [comment]
I don't like that display either and googling for a solution brought me here. The image below shows what I did for a solution, but I am wondering if there is a better way. I'd also like to get rid of the double quotes. 
<img width="729" alt="df" src="https://user-images.githubusercontent.com/6801861/103179118-abb3b280-4856-11eb-9759-85301da2ab2b.png">

**Comment by gszep** [comment]
Happy holidays to you too! :smile: it would be nice to only show the string value without the type information and without quotations

**Comment by rleyvasal** [comment]
I agree the quotes should not be displayed on CategoricalValue, but the type should still be included in the output. The variable type is very useful specially when merging datasets  with different data types.

Julia displays the type under the variable name when listing a table in Jupyter notebooks (This would be the best solution).
![JupyterCategoricaltype](https://user-images.githubusercontent.com/61335136/103298107-80dc7200-49ae-11eb-93d1-c847468c95b7.PNG)

By comparison, R Studio also displays the type under the variable name ( <fctr> for factor and <int> for integer) - without quotes 
![DisplayCategoricalVariables](https://user-images.githubusercontent.com/61335136/103296634-50470900-49ab-11eb-9ff5-06bafc8b9024.PNG)

**Comment by greimel** [comment]
Here is a fix,

```julia
Base.show(io::IO, ::MIME"text/html", x::CategoricalArrays.CategoricalValue) = print(io, get(x))
```

before:
<img width="732" alt="image" src="https://user-images.githubusercontent.com/6280307/103447633-081b3580-4c8e-11eb-9963-87fa20c9549f.png">

after:

<img width="740" alt="image" src="https://user-images.githubusercontent.com/6280307/103447653-43b5ff80-4c8e-11eb-8dbb-3265df23623f.png">

Note, that this matches how CategoricalArrays are shown in the REPL.

<img width="399" alt="image" src="https://user-images.githubusercontent.com/6280307/103447684-cb037300-4c8e-11eb-82cc-624895a4bd21.png">

I wonder if Pluto somehow interferes with the show method defined in CategoricalArrays?

cc @nalimilan

**Comment by nalimilan** [comment]
`show(::CategoricalValue)` gives exactly what is displayed above, i.e. `CategoricalValue{Int64, UInt32} ...`. But this printing isn't supposed to be used when printing arrays or tables. DataFrames uses `print` to render the contents of its columns, and arrays use `show(IOContext(io, :typeinfo=>eltype(column)), column[i])`, which avoids repeating the type information. It would make sense for Pluto to print the eltype of the column under its name, and then pass `typeinfo` when printing like arrays.

**Comment by fonsp** [comment]
I proposed a fix: https://github.com/JuliaData/CategoricalArrays.jl/pull/318

**Comment by fonsp** [comment]
@nalimilan Any suggestions?

**Comment by jerlich** [comment]
Btw, I accidentally found a workaround. If you display a dataframe interpolated in markdown, it shows as in the repl
```Julia
begin
    df =  DataFrame=(a = ["foo", "bar", b=[1,2])
    df.a = categorical(a)
   md"""
   $(df)
   """
end
```
